CMSgov/onemac
# Oy2 22809 - move additional info section

Story: https://qmacbis.atlassian.net/browse/OY2-22809
Endpoint: See github-actions bot comment

### Details

Move additional info section for main package to be below main content and above Rai Response section

### Changes

- Moved AdditionalInfoSection component from DetailView to DetailSection component

### Test Plan

1. Login as any user with view privs
2. Navigate to any detail package with an RAI response (SPA or Waiver)
example SPA: MD-22-2205-VM
example Waiver: MD-12958.R01.00
3. Verify additional info section now appears above the RAI Response section.
